packages/scripts/src/upgrade-tests/voting2/2_Verify.ts

```diff
@@ -34,50 +43,53 @@ async function main() {
   const ownableContractsToMigrate = await getOwnableContracts(networkId);
   const multiRoleContractsToMigrate = await getMultiRoleContracts(networkId);
 
-  const votingV2Address = process.env["VOTING_V2_ADDRESS"];
-  const governorV2Address = process.env["GOVERNOR_V2_ADDRESS"];
-
-  if (!votingV2Address) throw new Error("VOTING_V2_ADDRESS not set");
-  if (!governorV2Address) throw new Error("GOVERNOR_V2_ADDRESS not set");
+  checkEnvVariables();
 
   const finder = await getContractInstance<FinderEthers>("Finder");
-  const governor = await getContractInstance<GovernorEthers>("Governor");
-  const oldVoting = await getContractInstance<VotingEthers>("Voting");
+  const registry = await getContractInstance<RegistryEthers>("Registry");
   const votingToken = await getContractInstance<VotingTokenEthers>("VotingToken");
 
-  const votingV2Factory: VotingV2Ethers__factory = await getContractFactory("VotingV2");
-  const votingV2 = await votingV2Factory.attach(votingV2Address);
+  const governor = await getContractInstance<GovernorEthers>("Governor", process.env[OLD_CONTRACTS.governor]);
```

Comment:
If the env var is not set, `getContractInstance` defaults to the deployed address stored in the repo